modules/prompt/functions/async

```diff
@@ -1 +1 @@
-../external/pure/async.zsh
\ No newline at end of file
+../external/async/async.zsh
```

Comment:
Ahh, perhaps my [this comment](https://github.com/sorin-ionescu/prezto/pull/1385#issuecomment-317865353) about broken symlink causing `pure` to throw `missing async` errors is about this change. Can the "_real_" `async` please stand up?